projects/ngx-charts-on-fhir/src/lib/fhir-chart/range-selector/range-selector.component.ts

```diff
@@ -0,0 +1,79 @@
+import { Component, ElementRef, Input } from '@angular/core';
+import { MatDatepickerInputEvent } from '@angular/material/datepicker';
+import { Chart } from 'chart.js';
+import { DataLayer } from '../../data-layer/data-layer';
+import { DataLayerManagerService } from '../../data-layer/data-layer-manager.service';
+
+
+@Component({
+  selector: 'range-selector',
+  templateUrl: './range-selector.component.html',
+  styleUrls: ['./range-selector.component.css'],
+})
+export class RangeSelectorComponent {
+  layers?: DataLayer[];
+  maxDate: Date | string;
+  minDate: Date | string;
+  isMatGroupFocus : boolean = true;
+
+ constructor(private layerManager: DataLayerManagerService, private el: ElementRef) { 
```

Comment:
is the ElementRef still needed? I don't see it used anywhere

Reply:
No, ElementRef is not needed Now.